src/std/main.ab

```diff
@@ -176,6 +176,12 @@ pub fun create_dir(path: Text): Null {
     }
 }
 
+pub fun delete_dir(path: Text): Null {
+    if dir_exist(path) {
```

Comment:
this `if` is useless, `rm -f` will ignore an error if it happened. it would be useful to check if its actually a file and not a directory or something, since `rm -f` on a directory would fail

Reply:
There was the idea to add the `rm` as builtin on https://github.com/amber-lang/amber/issues/215

Reply:
> this `if` is useless, `rm -f` will ignore an error if it happened. it would be useful to check if its actually a file and not a directory or something, since `rm -f` on a directory would fail

``rmdir`` only delete folder, fail if is a file and fail if folder is not empty.

Reply:
and maybe if is just a wrapper of rmdir this function can named ``rmdir``